oxenc/endian.h

```diff
@@ -24,7 +26,21 @@ extern "C" {
 #endif
 
 namespace oxenc {
-
+#if (defined(_LIBCPP_VERSION) && _LIBCPP_VERSION >= 140000) || \
+        (defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE >= 11)
```

Comment:
Please use [C++ feature macros](https://en.cppreference.com/w/cpp/feature_test) for this sort of thing (`__cpp_lib_bit_cast` here) instead of specific lib versioning.

Reply:
done